doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,134 @@ Release management
 Release policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A Tarantool release is identified by three digits, for example, 1.7.7.
-We use these digits according to their definitions provided at http://semver.org:
+A Tarantool release is identified by three digits, for example, 1.10.7:
+
+* The first digit stands for a MAJOR release that introduces
+  some *major changes*. Up to now, there has been only one major release jump
+  when we delivered the 2.x release series with the SQL support.
+* The second digit stands for a MINOR release that is used for
+  introducing new *features*. Some of the features may have no backward
+  compatibility.
```

Comment:
> The second digit stands for a MINOR release that is used for introducing new *features*.

I would also add that we call it a release series. Say, add:

> [...] a MINOR releases (so called release series) [...]

'features without backward compatibility' sounds strange. If a feature was introduced in `x.y` and is not present in `x.(y-1)`, then the term 'backward compatibility' is not applicable here.

I would say about backward compatibility without context of a feature. Backward incompatible change may be introduced as part of refactoring, which reduces code complexity — not as part of a feature. I would rephrase it like:

> A release series may contain backward incompatible changes in the sense that existing Lua, SQL or C code that are run on a current release series may not be run with the same effect on a future series. However we don't exploit this rule and don't make incompatible changes without appropriate reason. We usually deliver information how mature a functionality is through release notes. Please, note that binary data layout is always compatible with a previous series as well as with LTS series (an `x.y` instance can be started on top of `x.(y+1)` or `1.10.z` data), binary protocol is compatible too (as client-server as well as replication protocol).

Hm, in appears to be large. Maybe it worth to just say that backward incompatible changes are possible between release series here and describe exact meaning of this somewhere below.

Let's consider the comment as source of information and rephrase it for better readability, where necessary.